### ISSUE TITLE
Mavic Air Support?

### ISSUE THREAD

**Issue by danammeansbear** [diux-dev/rosettadrone#36]
Is this working with the mavic air? i will be testing it tonight but are there any problems known to you at the moment?

**Comment by sjd69** [comment]
No, the current version does not. You can give this fork a shot if you'd like (no guarantee it works, I haven't used it): https://github.com/The1only/rosettadrone . 
The project is not actively being maintained anymore. Possibly down the road I'll give this project a rewrite and work on it further in my free time, but there's no current plans to continue development on this project.

**Comment by danammeansbear** [comment]
I will see what i can accomplish. i am only a level 1 programmer but who knows! is there documentation on how other made support for other drones? thank you so much for the quick reply and something cool to use for my own projects!

**Comment by The1only** [comment]
I have not tried the Air on the new built. I will do that later this spring. If you do, let me know how it vent.  Use master on The1only site, it might work as it uses all the latest modules. Right now I am addind the DJI simulator to enable easier testing. Yesterday I uppdated the Mavlink version to the latest. Best regards The1only

**Comment by danammeansbear** [comment]
I was able to build successfully but I think due to me previously working
on unity android projects I have to re install android studio and my sdk.
Will let you know how it goes later today. Thank you for the response and
opportunity to work with you.

**Comment by danammeansbear** [comment]
Would you have any reason why my app keeps saying Register SDK fails, check
network is connected? I made the keys.xml file in the values folder then
the build says it runs successfully but the app never connects.
*Adam Dabdoub*
Autonomy Systems Technician
Sensors & Software Systems Division

300 College Park, Dayton, OH 45469-0089
c: 513.886.0301|  udri.udayton.edu <https://www.udayton.edu/udri/>

*University of Dayton Research Institute*

**Comment by danammeansbear** [comment]
Missions work and waypoints work. Looking for another person with a mavic
air to test if swarming works with them. Thank you for contributers.

**Comment by The1only** [comment]
You need to insert youre dkj key in the keys.xml file. It should not have been checked inn. My mistake

**Comment by The1only** [comment]
You got the air connected? Or swapped to mavic?

**Comment by danammeansbear** [comment]
I got my air connected after reinstalling android and the android SDK. Had
to switch to android 8.1.1

**Comment by The1only** [comment]
And live video??? Is that working?

**Comment by danammeansbear** [comment]
Havent had a chance to get to this project in awhile. gonna revamp it so i can get swarming
